apps/web/pages/tasks.tsx

```diff
@@ -34,12 +34,19 @@ export default function Tasks(props) {
       }
 
       <AddTaskItem addTaskItem={addTaskItem} isLoading={isLoading} />
+      <Flex
+      overflowY="scroll"
+      sx = {{
+      '::-webkit-scrollbar' :{display: 'none'}
```

Comment:
Nice!

Reply:
I think it'd be nice if we had a consistent style guide.
Some strings here use double quotes and others use single quotes.